### PR TITLE
feat(jexl): allow usage of form meta in question jexl

### PR DESCRIPTION
**Pull request by anehx**

### CODE REVIEW

File: caluma/caluma_form/structure.py [hunk 1]
```diff
@@ -38,6 +38,10 @@ def root(self):
         return self
 
     def get(self, name, default=None):
+        for original, aliases in self.aliases.items():
```
Comment (sbor23) [inline caluma/caluma_form/structure.py:41]: `self.aliases` will only be defined for `Fieldset`. Not sure if the jexl will strictly only work on `FieldSet` but I would prefer either error handling here or having `aliases = {}` in `Element`.

**Review by sbor23 — CHANGES_REQUESTED** [review]

File: caluma/caluma_form/structure.py (hunk 1 above)
Comment (anehx) [reply caluma/caluma_form/structure.py:43]: Good point, fixed

**Review by sbor23 — APPROVED** [review]
Good job!

File: caluma/caluma_form/structure.py [hunk 2]
```diff
@@ -22,6 +22,8 @@ def new_method(self, *args, **kwargs):
 
 
 class Element:
+    aliases = {}
```
Comment (winged) [inline caluma/caluma_form/structure.py:25]: Is this a class property? If not, you should move it to `__init__()` instead

File: caluma/caluma_form/structure.py [hunk 3]
```diff
@@ -124,11 +130,25 @@ def children(self):
         ]
 
 
+class MetaObject(Element):
```
Comment (winged) [inline caluma/caluma_form/structure.py:133]: You could probably inherit from `Element, dict` so you don't have to proxy through to the "inner" dict.

Also, seeing as it doesn't actually do much - is the `Element` inheritance really required? Could you do with using a simple dict instead?

File: caluma/caluma_form/structure.py [hunk 4]
```diff
@@ -38,6 +40,10 @@ def root(self):
         return self
 
     def get(self, name, default=None):
+        for original, aliases in self.aliases.items():
```
Comment (winged) [inline caluma/caluma_form/structure.py:43]: I think if you flip the data structure here, you might be able to get rid of the loop and improve lookup speeds. If it's of the structure `name` -> `actual_name`, you could reduce the slow lookup loop into this:

```python
name = self.aliases.get(name, name)
```

**Review by winged — CHANGES_REQUESTED** [review]
Nice work, but needs some cleanups & optimisations. Not sure the meta thing needs to be an `Element`

**Review by winged — APPROVED** [review]

**Pull request merged**